Fix build failure with CFLAGS="-O0" and libc
messages.c:44:15: error: ‘LC_NUMERIC’ undeclared (first use in this function)

--- src/messages.c
+++ src/messages.c
@@ -14,6 +14,10 @@
 #include <time.h>
 #include "wbt201.h"
 
+#ifdef HAVE_LOCALE_H
+#include <locale.h>
+#endif
+
 static char *wmodes[] = {
     /*0,1*/	NULL,NULL,
     /*2,3*/	"low_speed_limit1", "high_speed_limit1",
